cassandra/deserializers.pyx

```diff
@@ -127,8 +127,8 @@ cdef class DesCounterColumnType(DesLongType):
 
 cdef class DesDateType(Deserializer):
     cdef deserialize(self, Buffer *buf, int protocol_version):
-        cdef double timestamp = unpack_num[int64_t](buf) / 1000.0
```

Comment:
I can see how switching to integer math could change precision, but what was the impetus here? It looks like the pure python version [starts with floating point math](https://github.com/datastax/python-driver/blob/3.0.0/cassandra/cqltypes.py#L558). Can you help me understand why this had to change?